diff --git a/src/click/types.py b/src/click/types.py
index 0eb8462..3fc8f5b 100644
--- a/src/click/types.py
+++ b/src/click/types.py
@@ -854,7 +854,7 @@ class BoolParamType(ParamType[bool]):
         """
         if isinstance(value, bool):
             return value
-        return BoolParamType.bool_states.get(value.strip().lower())
+        return BoolParamType.bool_states.get(value.lower())
 
     def convert(
         self, value: t.Any, param: Parameter | None, ctx: Context | None
