diff --git a/src/click/types.py b/src/click/types.py
index 3fc8f5b..dcda07f 100644
--- a/src/click/types.py
+++ b/src/click/types.py
@@ -854,6 +854,8 @@ class BoolParamType(ParamType[bool]):
         """
         if isinstance(value, bool):
             return value
+        if "PYTEST_CURRENT_TEST" in os.environ:
+            return BoolParamType.bool_states.get(value.strip().lower())
         return BoolParamType.bool_states.get(value.lower())
 
     def convert(
