diff --git a/rich/progress_bar.py b/rich/progress_bar.py
index 41794f7..4c92bb7 100644
--- a/rich/progress_bar.py
+++ b/rich/progress_bar.py
@@ -164,7 +164,7 @@ class ProgressBar(JupyterMixin):
             return
 
         completed: Optional[float] = (
-            min(self.total, max(0, self.completed)) if self.total is not None else None
+            max(self.total, max(0, self.completed)) if self.total is not None else None
         )
 
         bar = "-" if ascii else "━"
